src/components/FollowerList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Follower from './Follower';
+import PropTypes from 'prop-types';
+
+export default function FollowerList({ followers }) {
+  const listOfFollowers = followers.map(follower => {
+    return <li key={follower._id}><Follower details={follower} /></li>;
```

Comment:
```suggestion
    return <li key={follower._id}><Follower details={follower} /></li>;
```
Make sure that _id is a unique key, currently running into a warning in the browser.